tests/appsTest.js

```diff
@@ -0,0 +1,270 @@
+////////////////////////////////
+///////    MAIN TESTS    ///////
+////////////////////////////////
+
+/* globals ownCloud, __karma__ */
+
+describe("Currently testing apps management,", function() {
+// CURRENT TIME
+    var timeRightNow = new Date().getTime();
```

Comment:
if you import the "sinon" JS library, you can also use fake timers to reduce randomness of time-related tests